tests/unit/adaptors/graph/test_chains.py

```diff
@@ -234,7 +234,24 @@ def test_short_indirect_import(self):
 
         assert result == {("green.indirect", "purple", "blue.foo")}
 
-    def test_long_indirect_import(self):
+    def test_short_indirect_import_as_module(self):
+        graph = ImportGraph()
+        graph.add_module("green")
```

Comment:
Feels like we're adding a lot of modules unnecessarily in this test case - compare with `test_short_indirect_import_as_packages`?

Reply:
Yea, I've added some cases that are purposefully a bit longer to ensure we're getting the shortest ones we can. But I see this isn't the way other tests are done, can update quickly 👍🏼

Reply:
I've updated it so that we have one import of the blue module (green-purple-blue), and a shorter import of a module in the blue package (green-blue.foo) to ensure that we get the shortest path to a module if `as_packages` is False